packages/xo-web/src/xo-app/logs/backup-ng/log-alert-header.js

```diff
@@ -66,6 +67,23 @@ export default decorate([
       formattedLog: (_, { log }) => JSON.stringify(log, null, 2),
       jobFailed: (_, { log = {} }) =>
         log.status !== 'success' && log.status !== 'pending',
+      reportBugProps: ({ formattedLog }, { log = {} }) => {
+        const props = {
+          size: 'small',
+          title: 'Backup job failed',
+        }
+        if (currentPlan === XOA_PLAN_SOURCES) {
+          props.message = `\`\`\`json\n${formattedLog}\n\`\`\``
+        } else {
+          props.files = [
+            {
+              content: createBinaryFile(formattedLog),
+              name: `${ifDef(log.start, formatDate)} - backup.log`,
+            },
+          ]
```

Comment:
While we're at it, I think we should join the job configuration, what do you think?

Reply:
It seems so good to me :+1: